.github/workflows/ci.yml

```diff
@@ -183,3 +183,36 @@ jobs:
         run: |
           anchor test
         working-directory: ./contracts/solana/programs/stork
+
+  test-sui:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Rust
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Cache Rust dependencies
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target/
+          key: ${{ runner.os }}-cargo-sui-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-sui-
+
+      - name: Install Sui
```

Comment:
This looks like it took 33 minutes. Can this be cached?

Reply:
If not, is there a faster way to install this? At 33 minutes unfortunately don't think it will be practical to include this in the build.

Reply:
Caching worked. Job at around 1.5 minutes now